src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -301,8 +301,13 @@ def run(self, args, unknown_args):
         unknown_args += ('--tags=services',)
         rc = AnsiblePlaybook(self.parser).run(args, unknown_args)
         if ask("Would you like to update supervisor to use the new configurations?"):
-            exec_fab_command(args.env_name, 'supervisorctl:"reread"')
-            exec_fab_command(args.env_name, 'supervisorctl:"update"')
+            carryover_args = []
+            if args.limit:
+                carryover_args.extend(['--limit', args.limit])
+            commcare_cloud(
+                args.env_name, 'run-shell-command', 'webworkers:celery:pillowtop:formplayer',
```

Comment:
Short term: should this include the proxy and airflow? I think the websockets process runs on the proxy.

Long term: should this list of hosts be defined elsewhere. if there are any other additions to our groups, this may be a hard place to know to update

Reply:
Good points.

The full list being used by fab is https://github.com/dimagi/commcare-cloud/blob/a0e74dde3231c61ea4ad351e475ba837257155af/src/commcare_cloud/fab/const.py#L9-L18, which doesn't seem to include proxy, but that doesn't mean that's right. I'll make some changes in a followup PR

Reply:
staticfiles is the proxy https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/fab/fabfile.py#L296

Reply:
Oh right, thanks